### DISCUSSION TITLE
DevTools window is white

### DISCUSSION THREAD

**Discussion by bhosachin** [cefsharp/CefSharp#4378] [General]
we are using cefsharp 98 with WPF. Devtools window is opened white.

![image](https://user-images.githubusercontent.com/107455705/215091239-44b56ffd-710c-4a08-a260-afda0f556cf0.png)

[Logs](url) from debug.log file
[0127/173818.742:ERROR:CONSOLE(1)] "Uncaught (in promise) RangeError: Incorrect locale information provided", source: devtools://devtools/core/i18n/i18n.js (1)
we have local files in bin folder and we have not disabled devtools.

Its working with minimal example but somehow not with our application.
any clues will be really helpfull.

**Comment by amaitland** [comment]
I've not seen that particular error before. Are you setting `CefSettings.Locale`?
